### PR TITLE
"concepts/architecture.md"  translation

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

Relevant issue:https://github.com/open-cluster-management-io/OCM/issues/21

### CODE REVIEW

**Comment by qiujian16** [comment]
@yue9944882  anyone can help to review this?

**Comment by yue9944882** [comment]
@qiujian16 @xuezhaojun my apologies for missing the pr. the content of the architecture doc is recently refreshed. how do you think about translating from the latest version?

**Comment by xuezhaojun** [comment]
> @qiujian16 @xuezhaojun my apologies for missing the pr. the content of the architecture doc is recently refreshed. how do you think about translating from the latest version?

No problem, let me update it based on the latest.

**Comment by xuezhaojun** [comment]
The content has been updated, PTAL @yue9944882 @qiujian16

File: content/zh/concepts/architecture.md [hunk 1]
```diff
@@ -3,55 +3,107 @@ title: 架构
...
 {{< toc >}}
 
 <!-- spellchecker-enable -->
 
-## Overview
+## 概览
 
-open-cluster-management enables any capability within the Kubernetes ecosystem to orchestrate itself across multiple clusters and cloud providers. Consider the requirements that a capability within the Kubernetes ecosystem would need to become multicluster-aware.
+__Open Cluster Management__ (OCM) 是用于Kubernetes多集群编排的一个功能强大，模块化，可扩展的平台。
+通过吸取过去在Kubernetes社区构建联邦系统的失败教训，在OCM中，我们将跳出过去[Kubefed v2](https://github.com/kubernetes-sigs/kubefed)那种中心化，命令式的架构，并拥抱与Kubernetes中“master-kubbelet”原始模式一致的“master-agent”架构。
```
Comment (qiujian16) [inline content/zh/concepts/architecture.md:17]: kubbelet --> kubelet

Comment (qiujian16) [reply content/zh/concepts/architecture.md:17]: side topic, we should not use `master`.

Comment (xuezhaojun) [reply content/zh/concepts/architecture.md]: Ok, fixed.

**Comment by xuezhaojun** [comment]
The PR has been rebased to the latest. The word "master" was replaced. @qiujian16

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

Let's merge this at first, we still need to keep on refreshing the wording

**Pull request merged**